k6/health.js

```diff
@@ -0,0 +1,10 @@
+import http from 'k6/http';
+import { sleep } from 'k6';
+export const options = {
+  vus: 10,
+  duration: '30s',
+};
+export default function () {
+  http.get('http://0.0.0.0:3333/health');
```

Comment:
Codacy has a fix for the issue: [Replace `'http://0.0.0.0:3333/health'` with `"http://0.0.0.0:3333/health"`](https://app.codacy.com/gh/dasch-swiss/dsp-api/pullRequest?prid=10006168)
```suggestion
  http.get("http://0.0.0.0:3333/health");
```